libraries/plugins/witness/witness.cpp

```diff
@@ -143,7 +143,21 @@ void witness_plugin::plugin_startup()
 
 void witness_plugin::plugin_shutdown()
 {
-   // nothing to do
+   _shutting_down = true;
```

Comment:
I think this should be moved into ``stop_block_production()``

Reply:
@pmconrad Thanks ! Moved in new commit.